components/scheduler/templates/slot.md

```diff
@@ -46,6 +46,8 @@ The `context` of the template is a `SchedulerSlotTemplateContext` object that co
 ````CSHMTL
 @* Use the AllDaySlotTemplate and SlotTemplate *@
 
+@using Telerik.FontIcons
```

Comment:
Is this necessary? Customers should have it in `_Imports`. If REPL requires it, notify Kris to import the namespace by default.

Reply:
Ops, you are right.